tests/test_main.py

```diff
@@ -205,15 +205,15 @@ def test_post():
         assert response.status_code == 422, f'product = "" should return 422, got {response.status_code}'
 
         response = client.post("/product", headers=get_auth_token(), json=
-            {"product": "", "version": 0, "k": "test", "v": "test"})
+            {"product": "0000000000000", "version": 0, "k": "test", "v": "test"})
```

Comment:
This is not the same test !

Is was testing version=0 which should return an error.

Reply:
@cquest I thought for testing version, the product barcode must be a valid one. If its empty, pytest might directly assume that the error is due to the barcode, and not due to the version. Is my intiution right?
Thanks for helping me out!

Reply:
My bad, you're right. Sorry.

Copy/paste error in the tests... the first one should remain empty to check empty barcode, then 0000 for the others.

Reply:
> Copy/paste error in the tests... the first one should remain empty to check empty barcode, then 0000 for the others.

That's what I figured. I can leave the tests as is, right? (except the > 13 digits)